pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -209,10 +218,7 @@ void processArguments() throws Exception {
         protected String functionName;
         @Parameter(names = "--className", description = "The function's class name")
         protected String className;
-        @Parameter(
-                names = "--jar",
-                description = "Path to the jar file for the function (if the function is written in Java)",
-                listConverter = StringConverter.class)
+        @Parameter(names = "--jar", description = "Path to the jar file for the function (if the function is written in Java). It also supports url-path (http/https/file) from which worker can download the package.", listConverter = StringConverter.class)
```

Comment:
While http/https is clear, what does a --jar=file:<value> signifiy? Is it the same as --jar=<filename> or something else?

Reply:
umm.. `http` and `file` are protocol which can be supported by providing url. I will add clarification text to description for file-protocol.